scripts/test-acceptance.sh

```diff
@@ -53,4 +53,4 @@ if [[ "${test_name}" = "accessibility" ]]; then
 fi
 
 echo "Running acceptance tests from ${test_name}.py against the sandbox..."
-nosetests ../acceptance/${test_name}.py --with-xunit --xunit-file=../acceptance/xunit-${test_name}.xml
+nosetests ../acceptance/${test_name}.py:MultipleOpenAssessmentTest --with-xunit --xunit-file=../acceptance/xunit-${test_name}.xml
```

Comment:
It's not a hard-and-fast rule, but if we're adding new functionality that isn't tested in accessibility tests already, we should be adding a test to verify it.

In this case, I'd recommend adding one similar to https://github.com/edx/edx-ora2/blob/master/test/acceptance/accessibility.py#L217, but pointed at your new problem.